resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -5380,6 +5380,934 @@ vars.put(&quot;utcTime&quot;, utcTime);</stringProp>
           <hashTree/>
         </hashTree>
       </hashTree>
+      <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Auth Type Callback (Execution) " enabled="true">
+        <stringProp name="ThreadGroup.on_sample_error">continue</stringProp>
+        <elementProp name="ThreadGroup.main_controller" elementType="LoopController" guiclass="LoopControlPanel" testclass="LoopController" testname="Loop Controller" enabled="true">
+          <boolProp name="LoopController.continue_forever">false</boolProp>
+          <stringProp name="LoopController.loops">1</stringProp>
+        </elementProp>
+        <stringProp name="ThreadGroup.num_threads">1</stringProp>
+        <stringProp name="ThreadGroup.ramp_time">1</stringProp>
+        <boolProp name="ThreadGroup.scheduler">false</boolProp>
+        <stringProp name="ThreadGroup.duration"></stringProp>
+        <stringProp name="ThreadGroup.delay"></stringProp>
+        <boolProp name="ThreadGroup.same_user_on_next_iteration">true</boolProp>
+      </ThreadGroup>
+      <hashTree>
+        <CSVDataSet guiclass="TestBeanGUI" testclass="CSVDataSet" testname="Load Request Id From File" enabled="true">
+          <stringProp name="delimiter">,</stringProp>
```

Comment:
Masterdata Location Hierarchy Levels API - Any country can use more than one language code, can we take this comma separated in the user defiend variable and loop to use it ?

Reply:
We only have 3 language code in the DB i.e. eng,fra,ara so have added those in that file.

Reply:
When someone is using our scripts in their environment, it can have a single language or five languages as per setup. We have to configurable, keeping in the user variable will help with that

Reply:
I can see it is still not in the user variables, rather it is in some new user parameters inside the thread group. Not possible to take it comma separated in user variables, use a bean shell processor to split and consider the array ?

Reply:
Okay @vishwa-vyom will update this in another PR. For now have removed this from the current PR.

Reply:
Can you share a JIRA id that will track this work, Pragya also needs to be aware about this jira id to track.

Reply:
https://mosip.atlassian.net/browse/MOSIP-29024

Reply:
Please remove the jira id from PR header

Reply:
Done